core/src/main/scala/org/broadinstitute/dsde/rawls/fastpass/FastPassService.scala

```diff
@@ -296,6 +299,14 @@ class FastPassService(protected val ctx: RawlsRequestContext,
     dataSource
       .inTransaction { implicit dataAccess =>
         for {
+          isMigrating <- dataAccess.multiregionalBucketMigrationQuery.isMigrating(workspace)
+          _ = if (isMigrating) {
+            throw new RawlsExceptionWithErrorReport(
```

Comment:
This won't cause Sentry noise, will it?

Reply:
From what I can tell it won't. This gets caught further down on [line 336](https://github.com/broadinstitute/rawls/blob/07a738b79fc4fa0c1d516d59ea1e5c38707016bc/core/src/main/scala/org/broadinstitute/dsde/rawls/fastpass/FastPassService.scala#L336) along with other FastPass failures. FastPass failures in general aren't getting sent to Sentry, so these won't either. If I'm wrong and these do end up getting sent to Sentry, it should be infrequent at the very least and we can always mute it if it gets too noisy.